api/src/apollo/Event/type.ts

```diff
@@ -9,25 +10,67 @@ export const Event = objectType({
     t.nonNull.string('name');
     t.nonNull.string('description');
     t.nonNull.string('poster');
-    t.nonNull.id('location');
     t.nonNull.date('startDate');
     t.nonNull.date('endDate');
-    t.nonNull.list.nonNull.id('orgID');
     t.nonNull.orgType('orgType');
     t.nonNull.list.nonNull.string('notes');
-    t.nonNull.list.nonNull.id('poc');
     t.nonNull.boolean('weekly');
     t.repeatType('repeatDay');
     t.nonNull.int('priority');
     t.nonNull.status('status');
+
+    t.nonNull.id('locationID');
+    t.field('location', {
+      type: 'Location',
+      resolve(parent, _args, { prisma }) {
+        return prisma.location.findUnique({
+          where: {
+            id: parent.locationID,
+          },
+        });
+      },
+    });
+
+    t.nonNull.list.nonNull.id('pocID');
+    t.nonNull.list.field('poc', {
+      type: 'User',
+      async resolve(parent, _args, { prisma }) {
+        const pocs: Array<NexusGenObjects['User']> = [];
+
+        const promiseCalls = parent.pocID.map(id =>
+          prisma.user
+            .findUnique({
+              where: { id },
+            })
+            .then(poc => {
+              if (poc) pocs.push(poc);
```

Comment:
Let's not do this way, use find in array of IDs way for all these types of resolvers